terminus/geometry/bounding_box.py

```diff
@@ -0,0 +1,33 @@
+from geometry.point import Point
+
+
+class BoundingBox(object):
+
+    def _normalize(self):
+        new_origin = Point(min(self.origin.x, self.corner.x),
```

Comment:
This is a typical case where it would be handy to just do `self.origin.min(self.corner)`, not only because then you wouldn't have to repeat a very similar code in `merge`, but because you could also write the test for it the `Point` test class and then be more confident of your code. The same applies for `max`